src/assignement2/NewUser.java

```diff
@@ -0,0 +1,68 @@
+package assignement2;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+/*
+ * This class is inputing all the parameter required
+ * for the User
+ */
+
+public class NewUser {
+
+    /*
+     * method is taking input of name
+     * roll no
+     * age
+     * address
+     * and the course he choices
+     * @return user class
+     */
+
+    public static User getNewUser() {
+        final Scanner sc = new Scanner(System.in);
+        ArrayList<String> subj = new ArrayList<>();
+        final int maxSub = Constants.ch_5;
+        System.out.println("Enter name");
+        String name = sc.next();
+        System.out.println("Enter roll no");
+        int rollno = sc.nextInt();
+        if (rollno < 0) {
+
+            throw new NumberFormatException();
+        }
+        System.out.println("Enter age");
+        int age = sc.nextInt();
+        if (age < 0) {
+
+            throw new NumberFormatException();
```

Comment:
no specific message, please give a specific message. Plus its check it can be a separate functionality.

Reply:
made changes and put appropriate message as well